open_schools_platform/utils/sms_provider_requests.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+import requests
+
+from open_schools_platform.common.constants import CommonConstants
+
+
+def prepare_phone_number(phone: str):
+    phone = str(phone)  # PhoneNumber cast
+
+    if phone[0] == '+':
+        return phone[1:len(phone)]
```

Comment:
What's about Uzbekistan +998?

Reply:
Look, @JENIA21 !

Reply:
why do we need an extra variable in the code (phone)?
And I'd rather use regulars